ee/server/service/teams.go

```diff
@@ -1523,6 +1523,9 @@ func unmarshalWithGlobalDefaults(b *json.RawMessage) (fleet.Features, error) {
 func (svc *Service) updateTeamMDMDiskEncryption(ctx context.Context, tm *fleet.Team, enable *bool) error {
 	var didUpdate, didUpdateMacOSDiskEncryption bool
 	if enable != nil {
+		if len(svc.config.Server.PrivateKey) == 0 {
```

Comment:
more idiomatic to use `== ""` for string evaluation

Reply:
I had the same thought, but this follows a pattern set by folks on MDM team in many other places like [here](https://github.com/fleetdm/fleet/blob/adc2b41b8a99facab14bc19070d7f64b20e3dbf2/ee/server/service/vpp.go#L447), [here](https://github.com/fleetdm/fleet/blob/adc2b41b8a99facab14bc19070d7f64b20e3dbf2/server/service/appconfig.go#L369), and [here](https://github.com/fleetdm/fleet/blob/adc2b41b8a99facab14bc19070d7f64b20e3dbf2/server/service/apple_mdm.go#L4277). You think it's worth changing all of those?

Reply:
After consulting with MDM, let's use `== ""` going forward, and update existing `len`-based checks as we come to them, no need to update now.